Gems/ROS2/Code/Source/Manipulator/MotorizedJointComponent.cpp

```diff
@@ -0,0 +1,275 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include "ROS2/Manipulator/MotorizedJointComponent.h"
+#include "AzFramework/Physics/Components/SimulatedBodyComponentBus.h"
+#include <AzCore/Component/Entity.h>
+#include <AzCore/Component/TransformBus.h>
+#include <AzCore/Serialization/EditContext.h>
+#include <AzFramework/Physics/RigidBodyBus.h>
```

Comment:
The MotorizedJointComponent.h can be included without the full package path by using the "" relative include syntax, but the AzFramework include should use angle brackets.
Also can you reorder it after the AzCore includes right below?
```suggestion
#include "MotorizedJointComponent.h"
#include <AzCore/Component/Entity.h>
#include <AzCore/Component/TransformBus.h>
#include <AzCore/Serialization/EditContext.h>
#include <AzFramework/Physics/Components/SimulatedBodyComponentBus.h>
#include <AzFramework/Physics/RigidBodyBus.h>
```